content/en/docs/ops/deployment/security-model/index.md

```diff
@@ -0,0 +1,218 @@
+This task requires the `NET_ADMIN`, `SYS_ADMIN`, and `NET_RAW` capabilities.
+Like the [Istio CNI Plugin](#istio-cni-plugin), these are **node-local** privileges only.
+The Ztunnel does not have any associated Kubernetes RBAC privileges.
+
+Ztunnel is authorized to request certificates for any Service Accounts of pods running on the same node.
+Similar to [kubelet](https://kubernetes.io/docs/reference/access-authn-authz/node/), this explicitly does not allow requesting arbitrary
+certificates.
+This, again, ensures these privileges are **node-local** only.
+
+## Traffic Capture Properties
+
+When a pod is enrolled in the mesh, all incoming TCP traffic will be redirected to the proxy.
+This includes both mTLS/{{< gloss >}}HBONE{{< /gloss >}} traffic and plaintext traffic.
+Any applicable [policies](/docs/tasks/security/authorization/) for the workload will be enforced before forwarding the traffic to the workload.
+
+However, Istio does not currently guarantee that _outgoing_ traffic is redirect to the proxy.
+See [traffic capture limitations](/docs/ops/best-practices/security/#understand-traffic-capture-limitations).
+As such, care must be taken to follow the [securing egress traffic](/docs/ops/best-practices/security/#securing-egress-traffic) steps if outbound policies are required.
+
+## Mutual TLS Properties
+
+[Mutual TLS](/docs/concepts/security/#mutual-tls-authentication) provides the basis for much of Istio's security posture.
+Below explains various properties mutual TLS provides for the security posture of Istio.
+
+### Certificate Authority
+
+Istio comes out of the box with its own Certificate Authority.
+
+By default, the CA allows authenticating clients based on:
+* A Kubernetes JWT token, with an audience of `istio-ca`, verified with a Kubernetes `TokenReview`. This is the default method in Kubernetes Pods.
+* An existing mutual TLS certificate.
+* Custom JWT tokens, verified using OIDC (requires configuration).
+
+The CA will only issue certificates that are requested for identities that a client is authenticated for.
+
+Istio can also integrate with a variety of third party CAs; please refer to any of their security documentation for more information on how they behave.
+
+### Client mTLS
+
+{{< tabset category-name="dataplane" >}}
+{{< tab name="Sidecar mode" category-value="sidecar" >}}
+In sidecar mode, Istio client's will [automatically use TLS](/docs/ops/configuration/traffic-management/tls-configuration/#auto-mtls) when connecting to a service
+that is detected to support mTLS. This can also be [explicitly configured](/docs/ops/configuration/traffic-management/tls-configuration/#sidecars).
+Note that this automatic detection relies on Istio associating the traffic to a Service.
+[Unsupported traffic types](/docs/ops/configuration/traffic-management/traffic-routing/#unmatched-traffic) or [configuration scoping](/docs/ops/configuration/mesh/configuration-scoping/) can prevent this.
```

Comment:
Do you mean prevent the automatic detection from working properly?

Reply:
Yes there are some cases where it will not do mTLS when it would be able to

Reply:
Should we call out HBONE and how it solves issues like this, port confusion and server-sends first etc ? Can then say that it is enabled by default in ambient and opt-in for sidecar ?

Reply:
It could be relevant as we have heard people saying "security gotta love this one port 15008 only" so it does reduce port confusion and needs to open up a bunch of ports.